obspy/io/sac/header.py

```diff
@@ -22,7 +22,7 @@
                   when read into sac. [required]''',
        'b': 'F    Beginning value of the independent variable. [required]',
        'e': 'F    Ending value of the independent variable. [required]',
-       'iftype': '''I    Type of file [required]:
+       'iftype': '''I    Type of file [required]:\n
```

Comment:
Maybe it'd worth a shot to do something similar to `obspy-runtests`, i.e. add the possibility to build only one specific module..